spring-cloud-app-broker-integration-tests/src/test/java/org/springframework/cloud/appbroker/integration/fixtures/CloudControllerStubFixture.java

```diff
@@ -183,6 +183,12 @@ public void stubAppDoesNotExist(final String appName) {
 				.withBody(cc("empty-query-results"))));
 	}
 
+	public void stubListStacks() {
+		stubFor(get(urlPathEqualTo("/v2/stacks"))
```

Comment:
Can we avoid using the dead v2 API for new code?

Reply:
We are not explicitly calling stacks, it is the CF Java client during the push that validates the stacks and it uses that API call. Not much we can do without updating CFJC.

Reply:
Oh right... CFJC's `CloudFoundryClient` has `StacksV3 stacksV3()`, but maybe the Operations API is still going via V2?

Reply:
Exactly that.
Adding a note to review the v2 deprecation policy to see if we should start thinking about an OperationsV3.